google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugGlobalPollerTimerTask.java

```diff
@@ -7,15 +9,18 @@
  */
 class CloudDebugGlobalPollerTimerTask extends TimerTask {
 
+  private final CloudDebugProcessStateCollector stateCollector;
   private CloudDebugGlobalPoller cloudDebugGlobalPoller;
 
   public CloudDebugGlobalPollerTimerTask(CloudDebugGlobalPoller cloudDebugGlobalPoller) {
     this.cloudDebugGlobalPoller = cloudDebugGlobalPoller;
+    stateCollector = ApplicationManager.getApplication()
```

Comment:
I feel like intellij uses a style where they define the `getInstance()` method in `CloudDebugProcessStateCollection` class and that just calls the `ApplicationManager.getApplication().getComp....`